app/views/users/new.html.erb

```diff
@@ -0,0 +1,52 @@
+<h1>Create a New User Form</h1>
+<!--
+<form accept-charset="UTF-8" action="/users" method="post">
```

Comment:
Its better written this way `<form action=users_path method='post' accept-charset="UTF-8">`